atest/robot/standard_libraries/operating_system/modified_time.robot

```diff
@@ -8,7 +8,7 @@ ${TESTFILE}       %{TEMPDIR}${/}robot-os-tests${/}f1.txt
 *** Test Cases ***
 Get Modified Time As Timestamp
     ${tc} =    Check Test Case    ${TESTNAME}
-    Should Match Regexp    ${tc.kws[0].msgs[0].message}    Last modified time of '<a href=.*</a>' is 20\\d\\d-\\d\\d-\\d\\d \\d\\d:\\d\\d:\\d\\d
+    Should Match Regexp    ${tc.kws[0].msgs[0].message}    Last modified time of '<a href=.*</a>' is \\d\\d\\d\\d-\\d\\d-\\d\\d \\d\\d:\\d\\d:\\d\\d
```

Comment:
No, there're needed because with the ability to set the date all the way back to 1969 (epoch == 0), the date stamp prefix no longer is 20 and the tests would fail.

Reply:
That would require changing system time wouldn't it? Why would you ever want to do that when running Robot's acceptance tests?